packages/usage-examples/src/1-create-request-mock-storage.ts

```diff
@@ -0,0 +1,92 @@
+/**
+ * Creating a Request using mock storage
+ * =====================================
+ *
+ * This is an example of the simplest way to create a test Request, using a fake/mock storage.
+ * The request will be stored in memory and cleared as soon as the script is finished running.
+ */
+
+/**
+ * IMPORTS
+ *
+ * First we import the 2 packages we will need to create the request:
+ */
+// The signature provider allow us to sign the request
+import { EthereumPrivateKeySignatureProvider } from '@requestnetwork/epk-signature';
+// RequestNetwork is the interface we will use to interact with the Request network
+import * as RequestNetwork from '@requestnetwork/request-client.js';
+
+/**
+ * IDENTITY
+ *
+ * To create a request we need to declare the identities of the parties involved.
+ * Identities are the unique identifier of a request user. They are not payment addresses, only unique addresses that identify a person/entity.
+ */
+
+// Here we declare the payee identity,  with the payee identity ethereum address
+const payeeIdentity = {
+  type: RequestNetwork.Types.Identity.TYPE.ETHEREUM_ADDRESS,
+  value: '0x627306090abab3a6e1400e9345bc60c78a8bef57',
+};
+
+// Here we declare the (optional, but recommended) payer identity address.
+const payerIdentity = {
+  type: RequestNetwork.Types.Identity.TYPE.ETHEREUM_ADDRESS,
+  value: '0x740fc87Bd3f41d07d23A01DEc90623eBC5fed9D6',
+};
+
+/**
+ * SIGNATURE PROVIDER
+ *
+ * The ethereum private key signature provider allow a user to pass in their private ethereum key to sign a request.
+ * The signature is a proof of who created the request and of it's integrity (that no data changed after it was signed).
+ * This process is similar to the signature of an Ethereum transaction.
+ */
+
+// The signature info requires the request creator private key.
+// Please be careful with how you store and handle your private key, since it's a very sensitive piece of data.
+const payeeSignatureInfo = {
+  method: RequestNetwork.Types.Signature.METHOD.ECDSA,
+  privateKey: '0xc87509a1c067bbde78beb793e6fa76530b6382a4c0241e5e4a9ec0a0f44dc0d3',
+};
+
+const signatureProvider = new EthereumPrivateKeySignatureProvider(payeeSignatureInfo);
+
+/**
+ * REQUEST INFORMATION
+ *
+ * In the next section of code we declare the request information.
+ */
+
+// The main request info, with the currency, amount (in the smallest denominator), payee identity and payer identity
+const requestInfo: RequestNetwork.Types.IRequestInfo = {
+  currency: 'REQ',
+  expectedAmount: 1e18,
```

Comment:
I didn't know you could do that 😮

Reply:
We should have a string here. 
Plus, having a huge number like without using "BigNumber" is a risk to have operation issues.

Reply:
Javascript now has BigInts (notation: `1n`), don't they solve that issue?

Reply:
@benjlevesque they do, but also don't. They allow us to use big numbers, but they don't support scientific notation (so basically it's declaring a string to start with).